test-exports-cf/src/index.ts

```diff
@@ -37,16 +37,16 @@ export default {
     ctx: ExecutionContext
   ): Promise<Response> {
 
-    const constructorParameters
```

Comment:
Why is this changing?

Reply:
Ugh, sorry about that. I think my editor applied some Prettier rules I had configured for another repo.

I'll just disable that plugin and re-create this change.